Data/include/Poco/Data/TypeHandler.h

```diff
@@ -5674,6 +5674,7 @@ class TypeHandler<Poco::AutoPtr<T>>: public AbstractTypeHandler
 	/// Specialization of type handler for Poco::AutoPtr
 {
 public:
+        typedef AutoPtr<T> TPtr; // new code here
```

Comment:
please do not send unnecessary comments in PRs - it is obvious from the diff that his is new code

Reply:
will remove the obvious comment.